flocx_market/common/exception.py

```diff
@@ -0,0 +1,32 @@
+class MarketplaceException(Exception):
+    msg_fmt = "An unknown exception occurred."
+    code = 500
+    safe = False
+
+    def __init__(self, message=None, code=None, **kwargs):
```

Comment:
There's a problem with the current design.  If I have code like this:

```
try:
  raise MarketplaceException(code=400)
except MarketplaceException as err:
  print('Yikes, operation failed with error code {code}'.format(err.code))
```

I will get:

```
Yikes, operation failed with error code 500.
```

There are a couple ways of dealing with this. The simplest is just to drop the `code` keyword argument: if we need a custom response code, we create a new child exception.

Reply:
So if I drop the code arg then we just will have a custom child class for this. Which we do anyways for all of the errors currently being raised